CONTRIBUTING.md

```diff
@@ -154,4 +155,49 @@ See the [exe readme](./exe/README.md).
 
 ## Follow MATLAB style guidelines
 
-Please follow the style introduced in [MATLAB Guidelines 2.0, Richard Johnson](http://cnl.sogang.ac.kr/cnlab/lectures/programming/matlab/Richard_Johnson-MatlabStyle2_book.pdf).
\ No newline at end of file
+Please follow the style introduced in [MATLAB Guidelines 2.0, Richard Johnson](http://cnl.sogang.ac.kr/cnlab/lectures/programming/matlab/Richard_Johnson-MatlabStyle2_book.pdf).
+
+When making a pull request, the code is [checked](https://github.com/EcoExtreML/STEMMUS_SCOPE/actions/workflows/lint.yml) by the [MISS_HIT](misshit.org/) linter and style checker. 
+Any PR needs to pass this check before merging. 
+MISS_HIT is configured in `miss_hit.cfg`.
+
+For more details see the following instructions: 
+<details><summary>Installation instructions</summary>
+
+You need to have a valid python installation on your system.
+
+Create an enviroment with `venv` or conda:
+
+### **Venv**
+```bash
+python3 -m venv misshit # python on windows.
+```
+Activate this environment
+```bash
+source misshit/bin/activate
+```
+
+Or on Windows:
+```pwsh
+./misshit/Scripts/Activate.ps1
+```
+
+### **conda**
+```bash
+conda env create --name misshit
+conda activate misshit
+```
```

Comment:
This might be cumbersome for users. Is it possible to skip creating an environment?

Reply:
You can skip it, but it is a better practice.